src/components/fields/date-time-field/date-time-field.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: date-time-field
+title: Date-Time field
+permalink: /components/fields/date-time
+---
+
+import { IntlProvider } from 'react-intl';
+import { DateTimeField } from 'ui-kit';
+
+# DateTimeField
+
+A controlled date time input component for single date.
+
+## Usage
+
+```js
+import { DateTimeField } from '@commercetools-frontend/ui-kit';
+```
+
+## Example
+
+```.jsx-(minHeight:400px,maxHeight:400px)
+<IntlProvider locale="en">
+  <DateTimeField
+    title="Release Date"
+    value="2018-11-30T13:25:59.500Z"
+    timeZone="Europe/Berlin"
+    onChange={event => alert(event.target.value)}
+  />
+</IntlProvider>
+```
+
+#### Properties
```

Comment:
having props duplicated here and in the README is another reason why we should generate them. 🤔

Reply:
Yeah sure. Btw, this was basically just a dump of the readme 😅